src/main/scala/uk/gov/nationalarchives/StatusProcessor.scala

```diff
@@ -174,6 +175,17 @@ class StatusProcessor[F[_] : Monad](input: Input, allPuidInformation: AllPuidInf
       Status(res.fileId, FileType, statusName, statusValue)
     }).pure[F]
   }
+
+  private def zeroByteFFIDStatusHandling(disallowedReason: Option[String]): String = {
+    // 0 byte file cannot be password protected but in some instances PRONOM mis-identifies as password protected
+    // PRONOM uses the file extension to identify file type with 0 byte file.
+    // As a result for '.docx' extension PRONOM includes the encrypted type ('Microsoft Office Encrypted Document') as one of the identified types
+    if (disallowedReason.contains(PasswordProtected) || disallowedReason.isEmpty) {
```

Comment:
Do we need to check `disallowedReason.contains(ZeroByteFile)` as well?

Reply:
No, I think because:

- at the moment `ZeroByteFile` is not a disallowed reason (its actually inactive in the DB) so won't appear;
- if `ZeroByteFile`does at some point does become a valid disallowed reason then it would be caught by the default `else` statement anyway and will return it, and not `Success`

Does that make sense?

Reply:
Yeah, that makes sense.